### ISSUE TITLE
Insane amounts of trout in frozen ocean

### ISSUE THREAD

**Issue by BarchamMal** [BarchamMal/MC-Extended#22]
Which I have observed. and possibly other oceans.

**Comment by BarchamMal** [comment]
Also possibly rivers... testing now.

**Comment by BarchamMal** [comment]
It's in all biomes where trout spawn... I've fixed it... partially.

**Comment by BarchamMal** [comment]
This will stay opened even though I said it was fixed... it's not really fixed... no idea how to fix it.

**Comment by Raven-dreams** [comment]
I've noticed that they aren't just spawning in water,  I've been finding them inland in areas where a spawn source of water could be near by.  

As in we can be traveling around and then find random floating trout drops, or get to witness trout bouncing before they die and drop their item.  They are more commonly showing up on beaches or shores of large bodies of water, but have also witnessed floating trout drops while sailing a boat through an area.

I'm not sure where in the code the spawn information is located so I can't test out settings at this time.

**Comment by BarchamMal** [comment]
Right. this is a possibly different, but likely the same issue. I do not know what causes either of these, as I am only using fabric's mixing/spawning implementations. I may someday figure out how to fix this... but currently it's got me at a standstill.
